Automattic/jetpack
# Added project versioning suffixes.

This PR adds conditions to the build step that add suffixes to `alpha` versions based on the GitHub Action run ID. This will allow for the mirror repos to have package versions that are versioned with consecutive numbers.

## Proposed changes:
* Adds alpha suffixes to package versions.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:





I have tested the changes on [my fork of the repo](https://github.com/zinigor/jetpack/actions). You can re-run the build and download the artifacts to check the versions.
